tests/code/test_showUpcomingEvents.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from app.models.program import Program
+from app.models.interest import Interest
+from app.models.event import Event
+from app.models.user import User
+from app.logic.getUpcomingEvents import getUpcomingEventsForUser
+from peewee import DoesNotExist
+
+@pytest.mark.integration
+def test_getUpcomingEvents():
```

Comment:
Test isn't filled out enough. What happens on an error? Is it getting the correct events? This needs to be done in parallel with development on the function.